pkg/provisioner/controller.go

```diff
@@ -283,7 +281,7 @@ func (c *obcController) handleProvisionClaim(key string, obc *v1alpha1.ObjectBuc
 	// It is assumed that if the get claim fails, no resources were generated to begin with.
 	defer func() {
 		if err != nil {
-			log.Error(err, "cleaning up reconcile artifacts")
+			log.Info("provisioning errored, cleaning up API resources")
```

Comment:
`Info` instead of `Error` because Provision() errors are expected and handled?

Reply:
I'm channeling Dave Cheney here.  "Handle errors exactly once" - meaning either log them or return them.  I've spent a lot of time reading logs today and because it's both logged and returned, it's been difficult to tell from logs if it's occurring more than once.

Reply:
_"Handle errors exactly once"_ can be difficult sometimes but I agree it's a good philosophy,

Reply:
Agreed.  I think the line comes down to how ofter it appears in logs.  In this case, it was popping up 2 or 3 lines apart, several times before the reconcile iteration finished.